tensorflow/core/kernels/segment_reduction_ops.cc

```diff
@@ -332,7 +330,8 @@ class UnsortedSegmentSumOp : public UnsortedSegmentBaseOp<Device, T, Index> {
   explicit UnsortedSegmentSumOp(OpKernelConstruction* context)
       : UnsortedSegmentBaseOp<Device, T, Index>(
             context,
-           *(new functor::UnsortedSegmentSumFunctor<Device, T, Index>())) {}
+            sumFunctor) {}
+    functor::UnsortedSegmentSumFunctor<Device, T, Index> sumFunctor;
```

Comment:
I just joined gitter.im, could you look me up there? I'm available in the next hour, or after 4pm PST today.